src/write_outputs/write_vre_stor.jl

```diff
@@ -0,0 +1,429 @@
+		EndChargeACCap = existingcapchargeac[:] - retcapchargeac[:] + capchargeac[:],
+		StartDischargeDCCap = existingcapdischargedc[:],
+		RetDischargeDCCap = retcapdischargedc[:],
+		NewDischargeDCCap = capdischargedc[:],
+		EndDischargeDCCap = existingcapdischargedc[:] - retcapdischargedc[:] + capdischargedc[:],
+		StartDischargeACCap = existingcapdischargeac[:],
+		RetDischargeACCap = retcapdischargeac[:],
+		NewDischargeACCap = capdischargeac[:],
+		EndDischargeACCap = existingcapdischargeac[:] - retcapdischargeac[:] + capdischargeac[:]
+	)
+
+	if setup["ParameterScale"] ==1
+		dfCap.StartCapSolar = dfCap.StartCapSolar * ModelScalingFactor
+		dfCap.RetCapSolar = dfCap.RetCapSolar * ModelScalingFactor
+		dfCap.NewCapSolar = dfCap.NewCapSolar * ModelScalingFactor
+		dfCap.EndCapSolar = dfCap.EndCapSolar * ModelScalingFactor
+
+		dfCap.StartCapWind = dfCap.StartCapWind * ModelScalingFactor
+		dfCap.RetCapWind = dfCap.RetCapWind * ModelScalingFactor
+		dfCap.NewCapWind = dfCap.NewCapWind * ModelScalingFactor
+		dfCap.EndCapWind = dfCap.EndCapWind * ModelScalingFactor
+
+		dfCap.StartCapDC = dfCap.StartCapDC * ModelScalingFactor
+		dfCap.RetCapDC = dfCap.RetCapDC * ModelScalingFactor
+		dfCap.NewCapDC = dfCap.NewCapDC * ModelScalingFactor
+		dfCap.EndCapDC = dfCap.EndCapDC * ModelScalingFactor
+
+		dfCap.StartCapGrid = dfCap.StartCapGrid * ModelScalingFactor
+		dfCap.RetCapGrid = dfCap.RetCapGrid * ModelScalingFactor
+		dfCap.NewCapGrid = dfCap.NewCapGrid * ModelScalingFactor
+		dfCap.EndCapGrid = dfCap.EndCapGrid * ModelScalingFactor
+
+		dfCap.StartEnergyCap = dfCap.StartEnergyCap * ModelScalingFactor
+		dfCap.RetEnergyCap = dfCap.RetEnergyCap * ModelScalingFactor
+		dfCap.NewEnergyCap = dfCap.NewEnergyCap * ModelScalingFactor
+		dfCap.EndEnergyCap = dfCap.EndEnergyCap * ModelScalingFactor
+
+		dfCap.StartChargeACCap = dfCap.StartChargeACCap * ModelScalingFactor
+		dfCap.RetChargeACCap = dfCap.RetChargeACCap * ModelScalingFactor
+		dfCap.NewChargeACCap = dfCap.NewChargeACCap * ModelScalingFactor
+		dfCap.EndChargeACCap = dfCap.EndChargeACCap * ModelScalingFactor
+
+		dfCap.StartChargeDCCap = dfCap.StartChargeDCCap * ModelScalingFactor
+		dfCap.RetChargeDCCap = dfCap.RetChargeDCCap * ModelScalingFactor
+		dfCap.NewChargeDCCap = dfCap.NewChargeDCCap * ModelScalingFactor
+		dfCap.EndChargeDCCap = dfCap.EndChargeDCCap * ModelScalingFactor
+
+		dfCap.StartDischargeDCCap = dfCap.StartDischargeDCCap * ModelScalingFactor
+		dfCap.RetDischargeDCCap = dfCap.RetDischargeDCCap * ModelScalingFactor
+		dfCap.NewDischargeDCCap = dfCap.NewDischargeDCCap * ModelScalingFactor
+		dfCap.EndDischargeDCCap = dfCap.EndDischargeDCCap * ModelScalingFactor
+
+		dfCap.StartDischargeACCap = dfCap.StartDischargeACCap * ModelScalingFactor
+		dfCap.RetDischargeACCap = dfCap.RetDischargeACCap * ModelScalingFactor
+		dfCap.NewDischargeACCap = dfCap.NewDischargeACCap * ModelScalingFactor
+		dfCap.EndDischargeACCap = dfCap.EndDischargeACCap * ModelScalingFactor
```

Comment:
Do `*=` for these ... or see the `columns_to_scale` in `load_generators_data.jl`